schema/geometry_schema.json

```diff
@@ -957,14 +1003,18 @@
 					]
 				},
 				"illuminance_setpoint": {
-					"description": "Setpoint for daylighting control (lux).",
+					"description": "Setpoint for daylighting control.",
 					"type": "number",
-					"default": 300
+					"default": 300,
+					"si_units": "lux",
+					"ip_units": "fc"
```

Comment:
@shorowit this used to be always lux, and it's now dependent on the overarching units. Wanted to make sure you saw this in case you want to have a discussion on it.